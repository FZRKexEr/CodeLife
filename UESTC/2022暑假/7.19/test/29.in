10
rnxjavqmvevkjakkeahexwuvctudujfwogvspotlbftatshnwynepacvfhxyeniggajehufjrgnwadfdxprlrutommzldjcuqkug
6
dqhkfnifmfdhaucjzmtownjkolurdijmkdlgshubeurpyscdsdkdkoouvzhqzmbkibrrwjvwrivrwefpwwzsjbcraumrpcjxgcwk
9
czvapxftlxiqvloqfpljeumczokoynzgqgprftwjtvwkntqbteakslrjvofeplzrnnzvxrbwxhzfmtasbnreflmexxsdmewhbppq
10
sdtiaskwcnxjkqdhixmjumlpokaonaupkfyugilclxuutgmbemprnwrduypqvtmtlccbbkgpwwnplxwjnvlygyunsmpuswtbpvoa
1
hxfcslmyaozvjszfqyxuoxsycoqopmawuwwubjhxcghmmzcrxnasnnojufscbaqrzpgfyexdpwhavbxdwoegduyhsjyamzribytw
10
byixksujyemmrcdjdsuspcdhxrbryauryeywmtppmmbvtrgnpabcdynbonjrixedwpuonsdumyznanlakchedamtotxdjazoznyk
2
mbgwyvktjcjqcllolthkbbsmnjolitkzhptblfbcigfradncfzgtkqrfucesbjiedwbglbhbpmjmjvfdcacsueaitawuyfejbjai
6
yveurtvppqfzwzyeuhmsznsmhmgxhcsuareudswkmlcdwsnyscsgselrhxdogkrnbxjbftfevxxroaijqqiqppojhqurjqfxgkpp
6
abycwvxaahsglhxpyogpdfloqjyzpphjprmmtndhoypdeirsjzhmruesfvmjcbvllrdqwoesouzyuzrcvhbkaeclfkvjanmdbtff
7
vcvmcsmqljstwqhwvhfrqwkrdxmcgmdeuyelixpnbwhnqzyqtwysfxqecbrtnvkogihergfgeazhmompzjmzunvasnfpwnxqdvzq
6
