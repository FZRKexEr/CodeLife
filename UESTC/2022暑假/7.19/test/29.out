65536
2048
1024
882499718
1024
65980984
65536
65536
16384
65536
